CONTRIBUTING.md

```diff
@@ -226,31 +269,50 @@ $ git submodule update --recursive
 - Removing or renaming an existing intrinsic from the core module.
 
+### Source Code Formatting
+When the PR contains source code changes, one of the workflows will check the formatting of the code.
+
+Code formatting can be automatically fixed on your branch by commenting `/format`; a bot will proceed to open a PR targeting *your* branch. You can merge the generated PR into your branch, and the problem will be resolved.
+
+### Document Changes
+When the PR contains document changes for the [Slang User's Guide](https://shader-slang.com/slang/user-guide/), you need to update the table of contents by running a PowerShell script as follows:
+```
+# Open PowerShell on Windows
+cd docs
+.\build_toc.ps1
+
+# Add to git commit
+git add gfx-user-guide/toc.html
+git add user-guide/toc.html
+```
+
+Similar to the `/format` bot-command described in the previous section, you can also use `/regenerate-toc` instead.
+
+When the PR is limited to document changes, the build workflows may not start properly. This is because the building process is unnecessary when the PR is limited to document changes. This may lead to a case where some of the required build workflows are stuck waiting to start. When this happens, the committers will manually merge the PR as a workaround, and it will not give you a chance to rewrite the commit message.
 
 ## Code Style
-Follow our [Coding conventions](docs/design/coding-conventions.md) to maintain consistency throughout the project.
+Follow our [Coding Conventions](docs/design/coding-conventions.md) to maintain consistency throughout the project.
 
-Here are a few highlights
+Here are a few highlights:
 1. Indent by four spaces. Don't use tabs except in files that require them (e.g., Makefiles).
-1. Don't use the STL containers, iostreams, or the built-in C++ RTTI system.
-1. Don't use the C++ variants of C headers (e.g., use `<stdio.h>` instead of `<cstdio>`).
-1. Don't use exceptions for non-fatal errors (and even then support a build flag to opt out of exceptions).
-1. Types should use UpperCamelCase, values should use lowerCamelCase, and macros should use `SCREAMING_SNAKE_CASE` with a prefix `SLANG_`.
-1. Global variables should have a `g` prefix, non-const static class members can have an `s` prefix, constant data (in the sense of static const) should have a `k` prefix, and an `m_` prefix on member variables and a `_` prefix on member functions are allowed.
-1. Prefixes based on types (e.g., p for pointers) should never be used.
-1. In function parameter lists, an `in`, `out`, or `io` prefix can be added to a parameter name to indicate whether a pointer/reference/buffer is intended to be used for input, output, or both input and output.
-1. Trailing commas should always be used for array initializer lists.
-1. Try to write comments that explain the "why" of your code more than the "what".
+2. Don't use the STL containers, iostreams, or the built-in C++ RTTI system.
```

Comment:
Using "1." is proper markdown syntax so we don't have to manually keep track of the numbering.

Reply:
Right. I will bring it back.